### ISSUE TITLE
The information sidebar issue

### ISSUE THREAD

**Issue by kubalav** [fastaddons/SearchResultPreviews#5]
The results page is broken if there is an information sidebar.

[Broken page:](https://www.google.com/search?q=jan+werich)

![image](https://user-images.githubusercontent.com/37414585/198868922-f2e97692-2bd0-4de9-b4b1-1e593991be0e.png)

**Comment by fastaddons** [comment]
Thanks for the report! I'll look into it in the next release 2.6 this week.

**Comment by fastaddons** [comment]
OK I've looked into it, but it's slightly more complicated than I expected.
I will have to rebuild the "injection" so that the main content is not shifted, or resize the whole container, which then won't fit into the page on smaller screens...
I'll try to experiment but probably sometime next week since I need to finish the server asap.

**Comment by GrosBourrin** [comment]
same problem here

**Comment by GrosBourrin** [comment]
found a trick 

in userContent.css  (for firefox) add this :  

```
@-moz-document domain("google.fr") {
       * {             
                 --rhs-margin: 156px !important;                
       }
}
```

**Comment by Juraj-Masiar** [comment]
This should be now fixed in the new 4.3 version (available for Firefox, Chrome/Edge stores are reviewing the update).
